vhost-device-console/README.md

```diff
@@ -40,13 +43,20 @@ vhost-device-console --socket-path=<SOCKET_PATH>
  The localhost's port to be used for each guest, this part will be increased with
  0,1,2..socket_count-1.
 
--- option:: -b, --backend=nested|network
+-- option:: -b, --backend=nested|network|uds
 
   The backend type vhost-device-console to be used. The current implementation
-  supports two types of backends: "nested", "network" (described above).
+  supports 3 types of backends: "nested", "network", "uds".
   Note: The nested backend is selected by default and can be used only when
         socket_count equals 1.
 
+.. option:: -v, --uds-path=uds-file-path
+
+  The unix domain socket to be used for each guest, this path will be suffixed with
+  0,1,2..socket_count-1. e.g.: `--uds-path=/tmp/vm.sock --socket-count=2`
+  leads to two connectable vhost-user sockets:
+  /tmp/vm.sock0, /tmp/vm.sock1
```

Comment:
Please make it clear that this parameter makes sense on with `uds` backend

Reply:
ok